requirements.txt

```diff
@@ -4,12 +4,14 @@ Pypubsub==4.0.3
 configparser==5.0.1
 h5py==2.10.0
 imageio==2.9.0
+mido==1.2.10
```

Comment:
Remove `mido` and `python-rtmidi` since they will be in `optional-requirements.txt`

Reply:
Sorry, I thought I already did this -- apparently not. Thanks for the comment, it's done now.